Makefile

```diff
@@ -19,11 +19,12 @@ lint-fix: setup ## automatically fix linter errors (if possible)
 lint: setup ## run static analysis
 	npx eslint $(LINTED_FILES)
 
-test: setup ## run all tests
+test: setup ## run all tests, -w flag for watchmode tests for TDD iteration
+ifeq (,$(findstring w,$(MAKEFLAGS)))
```

Comment:
What do you think of having a single `ifeq` setting relevant vars that are later used in recipes? In that way we'd get rid of duplication at the cost of slightly higher complexity of something we don't change that often and should be only changed by people who know what they're doing. Thoughts?